crates/verify/src/sourcify.rs

```diff
@@ -47,155 +64,317 @@ impl VerificationProvider for SourcifyVerificationProvider {
                         args.address.to_string()
                     )?;
                     let response = client
-                        .post(args.verifier.verifier_url.as_deref().unwrap_or(SOURCIFY_URL))
+                        .post(&url)
                         .header("Content-Type", "application/json")
                         .body(serde_json::to_string(&body)?)
                         .send()
                         .await?;
 
                     let status = response.status();
-                    if !status.is_success() {
```

Comment:
nit: we can reorg this as
```rust
                    return match response.status() {
                        StatusCode::CONFLICT => {}
                        StatusCode::ACCEPTED => {}
                        _ => {}
```

also, won't the new code fail if 200 (as we're only handling 409 and 202) or doesn't matter?

Reply:
In the way the Sourcify api is implemented, the server will never return a 200 on this API call. I think it's fine to just expect 202.